backend/src/main/kotlin/blue/mild/covid/vaxx/dto/request/CreateVaccinationSlotsDtoIn.kt

```diff
@@ -27,5 +27,12 @@ data class CreateVaccinationSlotsDtoIn(
     /**
      * How long it takes to vaccinate single person.
      */
-    val durationMillis: Milliseconds
+    val durationMillis: Milliseconds,
+
+    /**
+     * First used queue id - when there are already "n" queues in the database
+     * for the given from/to time period and you want to use the same date interval,
+     * you need to set [queueOffset] to "n" to create new slots.
+     */
+    val queueOffset: Int = 0
```

Comment:
a kde se to v tom API nastavuje? To je body parameter?

Reply:
yes
![image](https://user-images.githubusercontent.com/14038818/120979049-769c7c00-c775-11eb-9035-26903daff22a.png)